.github/workflows/pr-commit-message-check.yml

```diff
@@ -0,0 +1,19 @@
+name: 'Commit Message Check'
```

Comment:
So eventually it looks like this on GH UI:
![image](https://user-images.githubusercontent.com/20625046/163389895-9acfb775-d06d-48fb-80ad-2af3ccaa9af6.png)

I'd suggest to rename it in a way that workflow name would be `Validations` or similar and the actual job name would be `PR Title`

Reply:
@yaronya is this what you meant? (see the PR checks).

Reply:
@TomerHeber I can't see it in the status checks now...

Reply:
@yaronya - you just need to change the title.
In any case, just did it. So you should see it now.

Reply:
@TomerHeber the name of the status check can't be dynamic since this way you cannot have it as required for the PR to pass.
I'd suggest to have it as `Validations / PR Title` since we might add more validations in the future

Reply:
@yaronya - modified.

Reply:
Thanks. 
It seems to fail now 🤔
Can you validate it please?

Reply:
@yaronya - where do you see a failure?

Reply:
IDK, but I don't see it now so 👍

Reply:
You probably saw it when I was testing (: